tests/integration/dynamic_ip_test.py

```diff
@@ -1902,3 +1902,31 @@ def test_set_dhcp_host_name_and_remove(dhcpcli_up):
     }
     libnmstate.apply(desired_state)
     assertlib.assert_state_match(desired_state)
+
+
+def test_auto_ip_with_pre_exist_address_without_dhcp_srv(eth1_up):
```

Comment:
Just a nit: could we make this test as tier1? Thanks!

Reply:
marked. Thanks!